test/synopsis.py

```diff
@@ -0,0 +1,40 @@
+import k3redisutil
+import time
+
+c = k3redisutil.RedisChannel(6379, '/foo', 'client')
+s = k3redisutil.RedisChannel(6379, '/foo', 'server')
+
+c.send_msg('c2s')

```

Comment:
Is this file a duplicate of `/synopsis.py`?

Reply:
sry!!my bad....